### PR TITLE
Create HelpWindowUtil class

### PR DESCRIPTION
**Pull request by tiewweijian**
Things that have been added in this commit
* Link to user guide from Help Window will open web browser
* Create new tests for HelpWindowUtil

### CODE REVIEW

File: src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java
Comment (charmainehly) [inline src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java:16]: Just a very minor suggestion, maybe the constants here can use `private static final` and make use of all upper case with `_` underscore for the constant name? :)

File: src/main/java/seedu/address/commons/util/HelpWindowUtil.java
Comment (domlimm) [inline src/main/java/seedu/address/commons/util/HelpWindowUtil.java:22]: Unable to test this, will KIV.

**Review by domlimm — CHANGES_REQUESTED** [review]
Good job @tiewweijian! Tough on you on fixing the codecov. Let's merge it first since `GUI Testing` can't be done right now.

Only comment similar to @charmainehly. Maybe you can make the change first. Then either of us approve and merge.

Thanks @tiewweijian! Good job! 💯

File: src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java
Comment (tiewweijian) [reply src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java:16]: missed that out! will change it in the next commit!

Comment (domlimm) [reply src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java:16]: wait @tiewweijian, would you want to add it in now? if not, we can approve and merge. your call!

Comment (domlimm) [reply src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java:16]: ok nvm. i think next one as this PR is linked to #41. charmaine created one i think you about it. let's approve this and merge and since charmaine gave 👍🏻, guess we can!

Comment (charmainehly) [reply src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java:16]: I think we can wait though? lets see what @tiewweijian says

Comment (tiewweijian) [reply src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java:16]: Mm I think I'll push a new commit over and then y'all can review, approve and merge ! So that its clearer that this PR is for issue 41 ! 
I'll close issue 41 once yall approve and merge in too !!

**Review by charmainehly — APPROVED** [review]
LGTM! :) thanks for making the changes!!

File: src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java
Comment (domlimm) [reply src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java]: @tiewweijian i'll tag the issue #41 to this PR. @charmainehly shall we approve and merge? you could do it! 😄

**Review by domlimm — APPROVED** [review]

**Pull request merged**
